docs-2.0/nebula-dashboard-en/1.what-is-dashboard-en.md

```diff
@@ -0,0 +1,47 @@
+# What is Nebula Dashboard Enterprise Edition
+
+Nebula Dashboard (Dashboard for short) is a visualization tool that monitors and manages the status of machines and services in Nebula Graph clusters. This topic introduces Dashboard Enterprise Edition. For details of Dashboard Community Edition, refer to [What is Nebula Dashboard Community Edition](../nebula-dashboard/1.what-is-dashboard.md).
+
+## Features
+
+- Create a Nebula Graph cluster of specified version, support import nodes in batch, add services with one click, and so on.
+
+- Support importing clusters, balancing data, scaling out or in, and so on through visualization interface.
```

Comment:
to uniform the style with all sentences, it's better to start the sentence with support, or delete support.

Reply:
`and so on` is not used often in technical writing. you can use etc or just delete it.